cmd/root.go

```diff
@@ -81,6 +99,31 @@ func configureLogrusHooks() error {
 	return nil
 }
 
+func configureHoneycombTelemetry() error {
+	honeycombEnv, err := loadHoneycombEnv()
+	if err != nil {
+		return err
+	}
+	// if env var not set, ignore
+	if honeycombEnv.SECRET_KEY == "" {
+		return nil
+	}
+	datasetName := honeycombEnv.DATASET_NAME
+	serviceName := honeycombEnv.SERVICE_NAME
+	if datasetName == "" {
```

Comment:
seems like envconfig has some sort of default tags you can use instead https://github.com/kelseyhightower/envconfig#struct-tag-support